scripts/kafka.setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BASEDIR=$(dirname $0)
+source $BASEDIR/kafka.conf
+[[ -f $BASEDIR/kafka.local.conf ]] && source $BASEDIR/kafka.local.conf
+
+pushd kafka
+  if [[ -f kafka.tgz ]]; then
+    echo "Kafka already downloaded"
+  else
+    echo "Downloading Kafka..."
+    curl -L -o kafka.tgz "https://www.apache.org/dist/kafka/$VERSION/kafka_$SVERSION-$VERSION.tgz"
+
+    echo "Extracting Kafka..."
+    tar -xzf kafka.tgz --strip 1
+  fi
+
+  echo "Formatting journal..."
+  bin/kafka-storage.sh format -t $UUID -c config/kraft/server.properties
+popd
+
+echo "All done, run ./kafka.start.sh"
```

Comment:
This is I think missing `test -d kafka || mkdir kafka` at the very beginning.

Reply:
We might even want to do test -d $BASEDIR/kafka || mkdir $BASEDIR/kafka ?

Reply:
Yep let me do that.